webapp/src/redux/ducks/studies.ts

```diff
@@ -37,6 +37,7 @@ export interface StudyFilters {
   users: Array<UserDTO["id"]>;
   groups: Array<GroupDTO["id"]>;
   tags: string[];
+  type: string;
```

Comment:
Move it after `archived`, rename it `variant` and change the type with boolean (because is a checkbox)